gateway-js/src/outOfBandReporter.ts

```diff
@@ -11,6 +15,18 @@ export const OUT_OF_BAND_REPORTER_QUERY = /* GraphQL */`#graphql
 
 const { name, version } = require('../package.json');
 
+type OobReportMutationResult =
```

Comment:
A little surprised that these types can't be easily created by graphql-code-generator, but looks like they can't be (yet). I guess those aren't the types returned via stuff like Apollo Client so it's not what is targeted.

Reply:
At the very least we could abstract this into a generic, not a bad idea. I think `GraphQLResponse` is a taken name but you get the idea.
`GraphQLResponse<OobReportMutationResult>`